pkg/platform/api/api.go

```diff
@@ -0,0 +1,201 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/logging"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"time"
+)
+
+const bottlerocketAPISock = "/run/api.sock"
+const minimumRequiredOSVer = "0.3.4"
+
+type updateState = string
```

Comment:
This is a [type alias](https://golang.org/doc/go1.9#language), which is used for migrating types.  You want a type definition instead for this use case.

```suggestion
type updateState string
```